import os

from PySide2 import QtCore, QtWidgets, QtGui

style = __file__.split('\\')[:-4]
style = os.path.join('/'.join(style), 'style.qss')

class Ui_Dialog(object):
    def setupUi(self, Dialog):

        with open(style, "r") as fh:
            Dialog.setStyleSheet(fh.read())

        path = __file__.split('\\')[:-4]
        path = "\\".join(path)
        path_resource = os.path.join(path, 'resource')

        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(Dialog.sizePolicy().hasHeightForWidth())
        Dialog.setSizePolicy(sizePolicy)
        Dialog.setMinimumSize(QtCore.QSize(300, 0))

        self.mainLayout = QtWidgets.QVBoxLayout(Dialog)
        self.mainLayout.setContentsMargins(10, 10, 10, 10)
        self.mainLayout.setSpacing(0)
        self.mainLayout.setObjectName("mainLayout")
        self.logo = QtWidgets.QLabel(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.logo.sizePolicy().hasHeightForWidth())
        self.logo.setSizePolicy(sizePolicy)
        self.logo.setMaximumSize(QtCore.QSize(16777215, 80))
        self.logo.setText("")
        self.logo.setTextFormat(QtCore.Qt.PlainText)
        self.logo.setPixmap(QtGui.QPixmap("{}\\logo.png".format(path_resource)))
        self.logo.setScaledContents(False)
        self.logo.setAlignment(QtCore.Qt.AlignCenter)
        self.logo.setWordWrap(False)
        self.logo.setObjectName("logo")
        self.mainLayout.addWidget(self.logo)
        self.renameLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.renameLayout.sizePolicy().hasHeightForWidth())
        self.renameLayout.setSizePolicy(sizePolicy)
        self.renameLayout.setObjectName("renameLayout")
        self.rename_Layout = QtWidgets.QHBoxLayout(self.renameLayout)
        self.rename_Layout.setContentsMargins(0, 5, 0, 0)
        self.rename_Layout.setSpacing(5)
        self.rename_Layout.setObjectName("rename_Layout")
        self.labelRename = QtWidgets.QLabel(self.renameLayout)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.labelRename.sizePolicy().hasHeightForWidth())
        self.labelRename.setSizePolicy(sizePolicy)
        self.labelRename.setMinimumSize(QtCore.QSize(60, 0))
        self.labelRename.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.labelRename.setObjectName("labelRename")
        self.rename_Layout.addWidget(self.labelRename)
        self.inputRename = QtWidgets.QLineEdit(self.renameLayout)
        self.inputRename.setFrame(False)
        self.inputRename.setObjectName("inputRename")
        self.rename_Layout.addWidget(self.inputRename)
        self.mainLayout.addWidget(self.renameLayout)
        self.startLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.startLayout.sizePolicy().hasHeightForWidth())
        self.startLayout.setSizePolicy(sizePolicy)
        self.startLayout.setObjectName("startLayout")
        self.start_Layout = QtWidgets.QHBoxLayout(self.startLayout)
        self.start_Layout.setSizeConstraint(QtWidgets.QLayout.SetDefaultConstraint)
        self.start_Layout.setContentsMargins(0, 5, 0, 5)
        self.start_Layout.setSpacing(5)
        self.start_Layout.setObjectName("start_Layout")
        self.labelStart = QtWidgets.QLabel(self.startLayout)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.labelStart.sizePolicy().hasHeightForWidth())
        self.labelStart.setSizePolicy(sizePolicy)
        self.labelStart.setMinimumSize(QtCore.QSize(60, 0))
        self.labelStart.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.labelStart.setObjectName("labelStart")
        self.start_Layout.addWidget(self.labelStart)
        self.inputStart = QtWidgets.QLineEdit(self.startLayout)
        self.inputStart.setFrame(False)
        self.inputStart.setObjectName("inputStart")
        self.start_Layout.addWidget(self.inputStart)
        self.labelPadding = QtWidgets.QLabel(self.startLayout)
        self.labelPadding.setObjectName("labelPadding")
        self.start_Layout.addWidget(self.labelPadding)
        self.inputPadding = QtWidgets.QLineEdit(self.startLayout)
        self.inputPadding.setFrame(False)
        self.inputPadding.setObjectName("inputPadding")
        self.start_Layout.addWidget(self.inputPadding)
        self.mainLayout.addWidget(self.startLayout)
        self.buttonRenameNumber = QtWidgets.QPushButton(Dialog)
        self.buttonRenameNumber.setAutoFillBackground(False)
        self.buttonRenameNumber.setStyleSheet("")
        self.buttonRenameNumber.setAutoDefault(False)
        self.buttonRenameNumber.setFlat(False)
        self.buttonRenameNumber.setObjectName("buttonRenameNumber")
        self.mainLayout.addWidget(self.buttonRenameNumber)
        self.removeLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.removeLayout.sizePolicy().hasHeightForWidth())
        self.removeLayout.setSizePolicy(sizePolicy)
        self.removeLayout.setObjectName("removeLayout")
        self.remove_Layout = QtWidgets.QHBoxLayout(self.removeLayout)
        self.remove_Layout.setContentsMargins(0, 5, 0, 0)
        self.remove_Layout.setSpacing(5)
        self.remove_Layout.setObjectName("remove_Layout")
        self.labelRemove = QtWidgets.QLabel(self.removeLayout)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.labelRemove.sizePolicy().hasHeightForWidth())
        self.labelRemove.setSizePolicy(sizePolicy)
        self.labelRemove.setMinimumSize(QtCore.QSize(62, 0))
        self.labelRemove.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.labelRemove.setFrameShadow(QtWidgets.QFrame.Plain)
        self.labelRemove.setLineWidth(1)
        self.labelRemove.setTextFormat(QtCore.Qt.RichText)
        self.labelRemove.setObjectName("labelRemove")
        self.remove_Layout.addWidget(self.labelRemove)
        self.buttonRemoveFirst = QtWidgets.QPushButton(self.removeLayout)
        self.buttonRemoveFirst.setAutoDefault(False)
        self.buttonRemoveFirst.setFlat(False)
        self.buttonRemoveFirst.setObjectName("buttonRemoveFirst")
        self.remove_Layout.addWidget(self.buttonRemoveFirst)
        self.buttonRemoveLast = QtWidgets.QPushButton(self.removeLayout)
        self.buttonRemoveLast.setAutoDefault(False)
        self.buttonRemoveLast.setObjectName("buttonRemoveLast")
        self.remove_Layout.addWidget(self.buttonRemoveLast)
        self.mainLayout.addWidget(self.removeLayout)
        self.hashLayout = QtWidgets.QWidget(Dialog)
        self.hashLayout.setEnabled(True)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.hashLayout.sizePolicy().hasHeightForWidth())
        self.hashLayout.setSizePolicy(sizePolicy)
        self.hashLayout.setObjectName("hashLayout")
        self.hash_Layout = QtWidgets.QHBoxLayout(self.hashLayout)
        self.hash_Layout.setContentsMargins(0, 5, 0, 5)
        self.hash_Layout.setSpacing(5)
        self.hash_Layout.setObjectName("hash_Layout")
        self.labelHash = QtWidgets.QLabel(self.hashLayout)
        self.labelHash.setObjectName("labelHash")
        self.hash_Layout.addWidget(self.labelHash)
        self.inputHash = QtWidgets.QLineEdit(self.hashLayout)
        self.inputHash.setFrame(False)
        self.inputHash.setObjectName("inputHash")
        self.hash_Layout.addWidget(self.inputHash)
        self.buttonHash = QtWidgets.QPushButton(self.hashLayout)
        self.buttonHash.setAutoDefault(False)
        self.buttonHash.setObjectName("buttonHash")
        self.hash_Layout.addWidget(self.buttonHash)
        self.mainLayout.addWidget(self.hashLayout)
        self.line = QtWidgets.QFrame(Dialog)
        self.line.setFrameShape(QtWidgets.QFrame.HLine)
        self.line.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line.setObjectName("line")
        self.mainLayout.addWidget(self.line)
        self.beforeLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.beforeLayout.sizePolicy().hasHeightForWidth())
        self.beforeLayout.setSizePolicy(sizePolicy)
        self.beforeLayout.setObjectName("beforeLayout")
        self.before_Layout = QtWidgets.QHBoxLayout(self.beforeLayout)
        self.before_Layout.setContentsMargins(0, 5, 0, 0)
        self.before_Layout.setSpacing(5)
        self.before_Layout.setObjectName("before_Layout")
        self.labelBefore = QtWidgets.QLabel(self.beforeLayout)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.labelBefore.sizePolicy().hasHeightForWidth())
        self.labelBefore.setSizePolicy(sizePolicy)
        self.labelBefore.setMinimumSize(QtCore.QSize(50, 0))
        self.labelBefore.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.labelBefore.setObjectName("labelBefore")
        self.before_Layout.addWidget(self.labelBefore)
        self.inputBefore = QtWidgets.QLineEdit(self.beforeLayout)
        self.inputBefore.setFrame(False)
        self.inputBefore.setObjectName("inputBefore")
        self.before_Layout.addWidget(self.inputBefore)
        self.buttonAddBeffore = QtWidgets.QPushButton(self.beforeLayout)
        self.buttonAddBeffore.setAutoDefault(False)
        self.buttonAddBeffore.setObjectName("buttonAddBeffore")
        self.before_Layout.addWidget(self.buttonAddBeffore)
        self.mainLayout.addWidget(self.beforeLayout)
        self.afterLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.afterLayout.sizePolicy().hasHeightForWidth())
        self.afterLayout.setSizePolicy(sizePolicy)
        self.afterLayout.setObjectName("afterLayout")
        self.after_Layout = QtWidgets.QHBoxLayout(self.afterLayout)
        self.after_Layout.setContentsMargins(0, 5, 0, 5)
        self.after_Layout.setSpacing(5)
        self.after_Layout.setObjectName("after_Layout")
        self.labelAfter = QtWidgets.QLabel(self.afterLayout)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.labelAfter.sizePolicy().hasHeightForWidth())
        self.labelAfter.setSizePolicy(sizePolicy)
        self.labelAfter.setMinimumSize(QtCore.QSize(50, 0))
        self.labelAfter.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.labelAfter.setObjectName("labelAfter")
        self.after_Layout.addWidget(self.labelAfter)
        self.inputAfter = QtWidgets.QLineEdit(self.afterLayout)
        self.inputAfter.setFrame(False)
        self.inputAfter.setObjectName("inputAfter")
        self.after_Layout.addWidget(self.inputAfter)
        self.buttonAddAfter = QtWidgets.QPushButton(self.afterLayout)
        self.buttonAddAfter.setAutoDefault(False)
        self.buttonAddAfter.setObjectName("buttonAddAfter")
        self.after_Layout.addWidget(self.buttonAddAfter)
        self.mainLayout.addWidget(self.afterLayout)
        self.line_2 = QtWidgets.QFrame(Dialog)
        self.line_2.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_2.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_2.setObjectName("line_2")
        self.mainLayout.addWidget(self.line_2)
        self.buttonsLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.buttonsLayout.sizePolicy().hasHeightForWidth())
        self.buttonsLayout.setSizePolicy(sizePolicy)
        self.buttonsLayout.setObjectName("buttonsLayout")
        self.buttons_Layout = QtWidgets.QHBoxLayout(self.buttonsLayout)
        self.buttons_Layout.setContentsMargins(0, 5, 0, 5)
        self.buttons_Layout.setSpacing(5)
        self.buttons_Layout.setObjectName("buttons_Layout")
        self.buttonGrp = QtWidgets.QPushButton(self.buttonsLayout)
        self.buttonGrp.setAutoDefault(False)
        self.buttonGrp.setObjectName("buttonGrp")
        self.buttons_Layout.addWidget(self.buttonGrp)
        self.buttonGeo = QtWidgets.QPushButton(self.buttonsLayout)
        self.buttonGeo.setAutoDefault(False)
        self.buttonGeo.setObjectName("buttonGeo")
        self.buttons_Layout.addWidget(self.buttonGeo)
        self.buttonCtrl = QtWidgets.QPushButton(self.buttonsLayout)
        self.buttonCtrl.setAutoDefault(False)
        self.buttonCtrl.setObjectName("buttonCtrl")
        self.buttons_Layout.addWidget(self.buttonCtrl)
        self.buttonJnt = QtWidgets.QPushButton(self.buttonsLayout)
        self.buttonJnt.setAutoDefault(False)
        self.buttonJnt.setObjectName("buttonJnt")
        self.buttons_Layout.addWidget(self.buttonJnt)
        self.buttonDrv = QtWidgets.QPushButton(self.buttonsLayout)
        self.buttonDrv.setAutoDefault(False)
        self.buttonDrv.setObjectName("buttonDrv")
        self.buttons_Layout.addWidget(self.buttonDrv)
        self.mainLayout.addWidget(self.buttonsLayout)
        self.line_3 = QtWidgets.QFrame(Dialog)
        self.line_3.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_3.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_3.setObjectName("line_3")
        self.mainLayout.addWidget(self.line_3)
        self.searchLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.searchLayout.sizePolicy().hasHeightForWidth())
        self.searchLayout.setSizePolicy(sizePolicy)
        self.searchLayout.setObjectName("searchLayout")
        self.search_Layout = QtWidgets.QHBoxLayout(self.searchLayout)
        self.search_Layout.setContentsMargins(0, 5, 0, 0)
        self.search_Layout.setSpacing(5)
        self.search_Layout.setObjectName("search_Layout")
        self.labelSearch = QtWidgets.QLabel(self.searchLayout)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.labelSearch.sizePolicy().hasHeightForWidth())
        self.labelSearch.setSizePolicy(sizePolicy)
        self.labelSearch.setMinimumSize(QtCore.QSize(60, 0))
        self.labelSearch.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.labelSearch.setObjectName("labelSearch")
        self.search_Layout.addWidget(self.labelSearch)
        self.inputSearch = QtWidgets.QLineEdit(self.searchLayout)
        self.inputSearch.setFrame(False)
        self.inputSearch.setObjectName("inputSearch")
        self.search_Layout.addWidget(self.inputSearch)
        self.mainLayout.addWidget(self.searchLayout)
        self.replaceLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.replaceLayout.sizePolicy().hasHeightForWidth())
        self.replaceLayout.setSizePolicy(sizePolicy)
        self.replaceLayout.setObjectName("replaceLayout")
        self.replace_Layout = QtWidgets.QHBoxLayout(self.replaceLayout)
        self.replace_Layout.setContentsMargins(0, 5, 0, 0)
        self.replace_Layout.setSpacing(5)
        self.replace_Layout.setObjectName("replace_Layout")
        self.labelReplace = QtWidgets.QLabel(self.replaceLayout)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Preferred)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.labelReplace.sizePolicy().hasHeightForWidth())
        self.labelReplace.setSizePolicy(sizePolicy)
        self.labelReplace.setMinimumSize(QtCore.QSize(60, 0))
        self.labelReplace.setAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
        self.labelReplace.setObjectName("labelReplace")
        self.replace_Layout.addWidget(self.labelReplace)
        self.inputReplace = QtWidgets.QLineEdit(self.replaceLayout)
        self.inputReplace.setFrame(False)
        self.inputReplace.setObjectName("inputReplace")
        self.replace_Layout.addWidget(self.inputReplace)
        self.mainLayout.addWidget(self.replaceLayout)
        self.optionsLayout = QtWidgets.QWidget(Dialog)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.optionsLayout.sizePolicy().hasHeightForWidth())
        self.optionsLayout.setSizePolicy(sizePolicy)
        self.optionsLayout.setObjectName("optionsLayout")
        self.options_Layout = QtWidgets.QHBoxLayout(self.optionsLayout)
        self.options_Layout.setContentsMargins(0, 5, 0, 5)
        self.options_Layout.setSpacing(5)
        self.options_Layout.setObjectName("options_Layout")
        self.optionHirearchy = QtWidgets.QRadioButton(self.optionsLayout)
        self.optionHirearchy.setChecked(True)
        self.optionHirearchy.setObjectName("optionHirearchy")
        self.options_Layout.addWidget(self.optionHirearchy)
        self.optionSelected = QtWidgets.QRadioButton(self.optionsLayout)
        self.optionSelected.setObjectName("optionSelected")
        self.options_Layout.addWidget(self.optionSelected)
        self.optionAll = QtWidgets.QRadioButton(self.optionsLayout)
        self.optionAll.setObjectName("optionAll")
        self.options_Layout.addWidget(self.optionAll)
        self.mainLayout.addWidget(self.optionsLayout)
        self.buttonApply = QtWidgets.QPushButton(Dialog)
        self.buttonApply.setAutoFillBackground(False)
        self.buttonApply.setStyleSheet("")
        self.buttonApply.setAutoDefault(False)
        self.buttonApply.setDefault(False)
        self.buttonApply.setFlat(False)
        self.buttonApply.setObjectName("buttonApply")
        self.mainLayout.addWidget(self.buttonApply)
        spacerItem = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
        self.mainLayout.addItem(spacerItem)

        widget = QtWidgets.QWidget()
        widget.setLayout(self.mainLayout)
        Dialog.layout().addWidget(widget)

        self.retranslateUi(Dialog)
        QtCore.QMetaObject.connectSlotsByName(Dialog)

    def retranslateUi(self, Dialog):
        _translate = QtCore.QCoreApplication.translate
        Dialog.setWindowTitle(_translate("Dialog", "Dialog"))
        self.labelRename.setText(_translate("Dialog", "Rename"))
        self.labelStart.setText(_translate("Dialog", "Start"))
        self.inputStart.setText(_translate("Dialog", "1"))
        self.labelPadding.setText(_translate("Dialog", "Padding"))
        self.inputPadding.setText(_translate("Dialog", "2"))
        self.buttonRenameNumber.setText(_translate("Dialog", "RENAME AND NUMBER"))
        self.labelRemove.setText(_translate("Dialog", "Remove..."))
        self.buttonRemoveFirst.setText(_translate("Dialog", "First Character-->"))
        self.buttonRemoveLast.setText(_translate("Dialog", "<--Last Character"))
        self.labelHash.setText(_translate("Dialog", "Hash rename"))
        self.inputHash.setText(_translate("Dialog", "name_####_suffix"))
        self.buttonHash.setText(_translate("Dialog", "Rename"))
        self.labelBefore.setText(_translate("Dialog", "Before"))
        self.inputBefore.setText(_translate("Dialog", "prefix_"))
        self.buttonAddBeffore.setText(_translate("Dialog", "Add"))
        self.labelAfter.setText(_translate("Dialog", "After"))
        self.inputAfter.setText(_translate("Dialog", "_suffix"))
        self.buttonAddAfter.setText(_translate("Dialog", "Add"))
        self.buttonGrp.setText(_translate("Dialog", "grp"))
        self.buttonGeo.setText(_translate("Dialog", "geo"))
        self.buttonCtrl.setText(_translate("Dialog", "ctrl"))
        self.buttonJnt.setText(_translate("Dialog", "jnt"))
        self.buttonDrv.setText(_translate("Dialog", "drv"))
        self.labelSearch.setText(_translate("Dialog", "Search"))
        self.labelReplace.setText(_translate("Dialog", "Replace"))
        self.optionHirearchy.setText(_translate("Dialog", "Hierarchy"))
        self.optionSelected.setText(_translate("Dialog", "Selected"))
        self.optionAll.setText(_translate("Dialog", "All"))
        self.buttonApply.setText(_translate("Dialog", "Apply"))